psplink/shell.c

```diff
@@ -1997,6 +1997,23 @@ static int remap_cmd(int argc, char **argv, unsigned int *vRet)
 	return CMD_OK;
 }
 
+// Iterative function to implement `atoi()` function in C
+static long atoi(const char* S)
```

Comment:
That's a funny question, this binary use kernel libraries so it doesn't use newlib functions, I suppose that main reason is to save space.

Why this function is now here, is because previously the toolchain had 2 different libc libraries, the newlib one, and another incomplete but smaller version.

That duplication of libraries has been removed now, and this is why atoi has been added here 🥲

Reply:
Ah, I didn't realize that. Makes sense. Might be good to remove the commented out include then.